File: app/src/main/java/com/vn/ezlearn/modelresult/LoginResult.java
package com.vn.ezlearn.modelresult;

/**
 * Created by FRAMGIA\nguyen.duc.manh on 10/10/2017.
 */

public class LoginResult {
    public Boolean success;
    public LoginData data;

    public static class LoginData {
        public String message;
        public Integer id;
        public String access_token;
        public String avatar;
        public String display_name;
    }
}

File: app/src/main/java/com/vn/ezlearn/modelresult/LoginResult.kt
package com.vn.ezlearn.modelresult

/**
 * Created by FRAMGIA\nguyen.duc.manh on 10/10/2017.
 */

class LoginResult {
    var success: Boolean = false
    var data: LoginData? = null

    class LoginData {
        var message: String = ""
        var id: Int? = null
        var access_token: String? = null
        var avatar: String? = null
        var display_name: String? = null
    }
}
